src/tasks/sep/050-op-betanet-add-game-type/config.toml

```diff
@@ -0,0 +1,31 @@
+templateName = "AddGameTypeTemplate"
+fallbackAddressesJsonPath = "src/tasks/sep/050-op-betanet-add-game-type/addresses.json"
+
+l2chains = [    
+    {name = "u18-beta-1", chainId = 420110022},
+]
+
+[[configs]]
+# The below addresses can be found here https://github.com/ethereum-optimism/devnets/blob/main/betanets/u18-beta/u18-beta-1/chain.yaml
```

Comment:
Why do we have to define the addresses here, if they are also in `addresses.json`? It would be great if we can remove them from here. And similar to above the addresses still don't match. For example this file has a systemConfig of `0x5babF93aC7AFfeaE25dc1a7ad4587fDeEbba44C2` but the linked file shows it as `0x4E0528A9dc40FEdf1C2cFc0eAA04e70E7d3d591B`